bin/hyperlog.js

```diff
@@ -185,20 +172,20 @@ const main = async () => {
     const client = new HyperCoreLogReader(storage, key, null, null, streamOpts)
 
     client.on('data', async (data) => {
-      let line = data.toString().trimRight()
-      if (multiFileLog) line = line.replace(inDir, '')
+      const { path, content } = HyperCoreFileLogger.parseLine(data.toString().trimRight())
 
-      if (logConsole) console.log(line)
+      if (logConsole) console.log(content)
 
       if (destination) {
-        if (multiFileLog) {
-          const { path, content } = HyperCoreFileLogger.parseLine(line)
+        if (path) {
```

Comment:
I think we cant do without an additional CLI parameter, since in order to find a shared path, the tool need to preprocess all the logs, and this can be a resource-intensive operation